src/libdbusmenuqt/dbusmenuimporter.cpp

```
@@ -455,6 +463,10 @@ void DBusMenuImporter::slotGetLayoutFinished(QDBusPendingCallWatcher *watcher)
             }
         } else {
             // Create
+            if (d->m_actionForId.count() >= MAX_TOTAL_ACTIONS) {
```

Clang (14.0.6)

[note] +3, including nesting penalty of 2, nesting level increased to 3

(clang)